nais/nais.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: "nais.io/v1alpha1"
+kind: "Application"
+metadata:
+  name: rekrutteringsbistand-stillingssok-proxy
+  namespace: arbeidsgiver
+  labels:
+    team: arbeidsgiver
+spec:
+  image: {{image}}
+  port: 8300
+  ingresses:
+  {{#each ingress as |url|}}
+      - {{url}}
+  {{/each}}
+  liveness:
+    path: /internal/isAlive
+  readiness:
+    path: /internal/isReady
+  strategy:
+    type: Recreate
+  replicas:
+    min: 1
+    max: 1
+  azure:
+    application:
+      enabled: true
+  elastic:
+    instance: rekrutteringsbistand-stilling
+  # Legg til envFrom: secret
+  env:
+    - name: JAVALIN_PORT
+      value: "8300"
```

Comment:
Har vi behov for å endre port noen gang? Er det ikke bare å hardkode en port i koden?

Reply:
Du har helt rett! Vi hardkoder!